modules/ols-0-2-1-release-notes.adoc

```diff
@@ -22,4 +22,6 @@ The following enhancements have been made with {ols-official} 0.2.1:
 
 * This release adds the ability for users to provide their own TLS certificates for communication between the {ols-long} console and the {ols-long} service.
 
-* With this release, the {ols-long} User Interface displays an email link that can be used to contact the {ols-official} Development Team.
\ No newline at end of file
+* With this release, the {ols-long} User Interface displays an email link that can be used to contact the {ols-official} Development Team.
+
+* This release introduces the ability to attach a YAML file to a question you ask on any page accessible from *Storage*, *Compute* or, *Builds* in the {ocp-product-title} web console.
```

Comment:
```suggestion
* This release introduces the ability to attach a YAML file to a question you ask on any page accessible from *Storage*, *Compute*, or *Builds* in the {ocp-product-title} web console.
```

Looks like the comma is misplaced by mistake. Once done, PR is ready for merge.

Reply:
oh dear lord, i quickly made this edit while on a meeting call and clearly introduced waste while doing so. 

thanks for catching it and for merging the PR @xenolinux!

Reply:
No problem :)